mne/epochs.py

```diff
@@ -498,15 +504,31 @@ def plot(self, epoch_idx=None, picks=None, scalings=None,
             Whether to halt program execution until the figure is closed.
             Useful for rejecting bad trials on the fly by clicking on a
             sub plot.
+        n_epochs : int
+            The number of epochs to show on a continuous plot. Defaults to 20.
+        n_channels : int
+            The number of channels per view on a continuous plot.
+            Defaults to 10.
+        fig_title : str | None
+            The title of the window. If None, epochs name will be displayed.
+            Defaults to None.
+        trellis : bool
+            Whether to use Trellis plotting. If False, the epochs are plotted
+            as continuous curves. Defaults to True.
```

Comment:
FWIW I'm also not a huge fan of calling the continuous plotting by setting Trellis == False. It seems more natural to me to set a flag as True when I want the function to take a specific behavior, not to set a flag to False when I want a function *not* to take on a specific behavior. It feels like the python argument version of a double-negative to me :P